### PR TITLE
fix(zkstack_cli): Apply zk_toolbox commands redesign

### PR DESCRIPTION
**Pull request by manuelmauro**
# Description

Apply zk_toolbox commands redesign.

### CODE REVIEW

**Comment by popzxc** [comment]
On hold until corresponding PR in core is merged & released

**Review by popzxc — DISMISSED** [review]

**Review by uF4No — DISMISSED** [review]

**Comment by uF4No** [comment]
Can you remove this callout from the quickstart, production and raas pages?

```
ZK Stack is still under development. We advise you to only use for local and testnet deployments.
```

**Comment by manuelmauro** [comment]
> Can you remove this callout from the quickstart, production and raas pages?
> 
> ```
> ZK Stack is still under development. We advise you to only use for local and testnet deployments.
> ```

Addressed in https://github.com/matter-labs/zksync-docs/pull/239/commits/e4e7bfe0688e6d6d8f8ef2662ab5c47bed9f043c

**Review by uF4No — APPROVED** [review]

**Pull request merged**
